layouts/shortcodes/issues-security.html

```diff
@@ -0,0 +1,18 @@
+<table>
+    <thead>
+      <tr>
+      <th>CVE ID</th>
+      <th>Summary</th>
+      <th>Issue Details</th>
+      </tr>
+    </thead>
+    <tbody>
+    {{ range $issues := getJSON "https://gist.githubusercontent.com/nehaLohia27/ffc2c57f0e32ab338d9f2a02b2fc9e7c/raw/431b5527613fca8c4b5bcff3ba1b06d92db47f88/issues" }}
```

Comment:
For the real piece, where will we fetch the JSON from?

Reply:
@sftim 
I have been trying to come up with ideas about where to place this. Would it make sense to create an automated PR workflow like this:
- Prow job runs periodically, and checks for additions in the vulnerabilities list
- A PR to `k/website` is created when a new vulnerability is added to the feed
- It goes through usual reviews and is then merged
- The docs page then shows up with the new vulnerability (JSON as well as Table)

Since the vulnerabilities are infrequent (handful of them per year) this flow could work and also allow PR process to pick up any mishaps or invalid inserts.

Reply:
I'd prefer to use more automation and avoid relying on approver availability.
There's a similar question about a place to put some JSON, for https://github.com/kubernetes/contributor-site/pull/222

So, although I don't know exactly how we usually hook up Prow job output to cloud object storage, there's the start of a pattern and a strong hint that a reusable answer (that uses automation) is worth chasing.

Reply:
@sftim I agree that cloud object storage will make approver availability a non-issue. But I am worried that the object storage without proper authorization and authentication could be a vector of compromise where the CVE data could be tampered with to deface k8s website or misrepresent the number of CVEs. 

Having the data (only when a meaningful delta exists, which is less than 10 times a year so far) go through approvals/reviews would not have such a problem. Perhaps this is an opportunity to use OWNERS file and allow SRC + SIG Security chairs and Tooling Lead (me) to be responsible for making sure this page on k/website, gets refreshed as soon as a new CVE is published. 

What do you think? @nehaLohia27 please chime in too if you have any thoughts on this!

Reply:
@PushkarJ @sftim what if we avoid using cloud object storage, then where will this json be placed ? I am still trying to understand the automated PR workflow process.  how would this PR know where to fetch the json from unless it is explicitly hosted somewhere? I need to understand this 4 steps below not clear on it yet. 

Prow job runs periodically, and checks for additions in the vulnerabilities list
An automated PR to k/website is created when a new vulnerability is added to the feed
It goes through usual reviews and is then merged
The docs page then shows up with the new vulnerability (JSON as well as Table)

Reply:
We now have two options to do this discussed in https://github.com/kubernetes/sig-security/issues/32 and https://github.com/kubernetes/sig-security/issues/33